transcom/mymove
# INT-B-18637 p2

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A904624&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Added missing implementation for Prime Queue and Prime move details AC 


### How to test

1. Create a move with BB Order Type
2. Login as a Prime
3. BB label should show in queue and on move details page
